.github/workflows/build.yaml

```
@@ -46,4 +49,4 @@ jobs:
         with:
           name: ${{ github.sha }}
           path: |
-            ${{ github.workspace }}/src/**/Release/*.nupkg
+            ${{ github.workspace }}/src/**/Release/*.nupkg
```

I think your editor is removing the empty line:

```suggestion
            ${{ github.workspace }}/src/**/Release/*.nupkg

```